CMakeLists.txt

```diff
@@ -108,11 +108,11 @@ if(WITH_ZMQ)
   endif()
 endif()
 
-tristate_option(WITH_USDT
-  "Enable tracepoints for Userspace, Statically Defined Tracing."
-  "if sys/sdt.h is found."
-  AUTO
-)
+option(WITH_USDT "Enable tracepoints for Userspace, Statically Defined Tracing." OFF)
```

Comment:
Do we want to have this OFF by default? Since v23.0 (specifically https://github.com/bitcoin/bitcoin/pull/23724), all release builds have had tracepoints compiled in by default as the systemtap headers are available.